kaldb/src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -69,6 +72,16 @@ public class Kaldb {
   protected MetadataStore metadataStore;
 
   Kaldb(KaldbConfigs.KaldbConfig kaldbConfig, S3Client s3Client) {
+    // Configure prometheus meter registry
+    prometheusMeterRegistry
+        .config()
+        .meterFilter(
+            new MeterFilter() {
+              @Override
+              public Meter.Id map(Meter.Id id) {
+                return id.withName(KALDB_METRICS_PREFIX + id.getName());
```

Comment:
What happens to metrics that we changed recently to manually have `kaldb_` prefix?